requirements/edx/base.in

```diff
@@ -84,7 +84,7 @@ edx-django-sites-extensions
 edx-django-utils>=5.0.0            # Utilities for cache, monitoring, and plugins
 edx-drf-extensions
 edx-enterprise
-edx-event-bus-kafka                 # Kafka implementation of event bus
+edx-event-bus-kafka>=0.6.2          # Kafka implementation of event bus
```

Comment:
Can we update the comment with why this constraint is here (i'm assuming it's so we can use the topic prefix?)

Reply:
Ah, yeah. We haven't been doing this in `.in` files, but should. Added.